tests/stp/configload/existTests/run.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+${TORQHOME}/torq.sh start all
```

Comment:
Do either of your set of tests need the full stack to be started? Or do you only need the stp to be started?

Reply:
I don't think so, only stp. just tested without startall and stopall and tests working ok. Will remove and push